graphviz.spec

```diff
@@ -43,16 +47,13 @@
 
 Summary:	Graph visualization tools
 Name:		graphviz
-Version:	2.38.0
-Release:	5
+Version:	20171130
```

Comment:
This is not a good idea because 20171130 > 2.39 etc. -- It will make it impossible to update to the next release version of graphviz without introducing a dreaded Epoch.
Let's go for Version: 2.38.1 Release: 0.20171130.1 the way we do elsewhere...

Reply:
I may have screwed up elsewhere with this. I may need a turorial on the epoch business.
Fixed for this package